migration.rst

```diff
@@ -0,0 +1,414 @@
+.. index::
+   single: Migration
+
+Migrating an Existing Application to Symfony
+============================================
+
+When you have an existing application that was not built with Symfony,
+you might want to move over parts of that application without rewriting
+the existing logic completely. For those cases there is a pattern called
+`Strangler Application`_. The basic idea of this pattern is to create a
+new application that gradually takes over functionality from an existing
+application. This migration approach can be implemented with Symfony in
+various ways and has some benefits over a rewrite such as being able
+to introduce new features in the existing application and reducing risk
+by avoiding a "big bang"-release for the new application.
+
+Prerequisites
+-------------
+
+The following steps do not require you to have the new Symfony
+application in place and in fact it might be safer to introduce these
+changes beforehand in your existing application.
+
+Before you start introducing Symfony to the existing application, you
+have to ensure certain requirements are met by your existing application.
+By extension this means that you will have to decide which version you
+are aiming to migrate to, either a current stable release or the long
+term support version (LTS). The main difference is, how frequently you
+will need to upgrade in order to use a supported version. Using the
```

Comment:
💡 With regards to legacy applications, there are more factors that influence this decision. For instance, the application might require legacy libraries that are not compatible with the most recent version of Symfony but work fine with LTS. Also, if the legacy application is pinned to an old php versions for some reason, you might chose to start with Symfony LTS and upgrade later.

Reply:
I have added the following sentence:
> In the context of a migration, other factors, such as the supported PHP-version or support for libraries/bundles you use, may have a strong impact as well.